docs/upgrade.html

```diff
@@ -33,6 +33,7 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                 <a href="/{{version}}/javadoc/org/apache/kafka/tools/api/RecordReader.html"><code>org.apache.kafka.tools.api.RecordReader</code></a>
                 interface to build custom readers for the <code>kafka-console-producer</code> tool.
             </li>
+            <li> Remove <code>use.incremental.alter.configs</code>. The modified behavior is identical to the previous <code>required</code> configuration </li>
```

Comment:
Could you add "therefore users should ensure that target broker is at least 2.3.0" to the docs